Makefile

```diff
@@ -20,17 +20,12 @@ IMPORT_OPTIONS := --vendor \
 		--repository=$(REPOSITORY) \
 		--name=$(NAME) \
 		--version=$(VER) \
-		--glob=**/*.yaml \
-		--ignore=dev \
+		--glob=resources/*.yaml \
```

Comment:
I think this might need to be `**/*.yaml` still, and `--ignore=dev` is still needed as well.